sys/libraries/mailer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+class mailer extends Library {
+
+	function __construct() {
+		$olddir = getcwd();
+		chdir(dirname(__FILE__));
+		require("PHPMailer/class.phpmailer.php");
+		//require('PHPMailer/PHPMailerAutoload.php');
+		chdir($olddir);
+		$this->mail = new PHPMailer;
+		$this->error = "";
+		$this->success = False;
+		$this->mail->isSMTP();
+		$this->mail->SMTPSecure = "tls";
+		$this->mail->Host = 'smtp.gmail.com';
+		$this->mail->Port = 587;
+		$this->mail->SMTPAuth = true;
+		$this->mail->Username = 'saathi.mailer@gmail.com';
+		$this->mail->Password = '';
```

Comment:
Correct. This can be added during deployment of the app. Currently I have made the above mentioned gmail id for testing. I have the password if anyone wants to test the feature.